src/settings/SettingsStore.ts

```diff
@@ -80,7 +80,7 @@ export const LEVEL_ORDER = [
 function getLevelOrder(setting: ISetting): SettingLevel[] {
     // Settings which support only a single setting level are inherently ordered
     if (setting.supportedLevelsAreOrdered || setting.supportedLevels.length === 1) {
-        return setting.supportedLevels;
+        return [...setting.supportedLevels];
```

Comment:
Might be worth adding a comment why we return a copy of this but not the other return